### PR TITLE
react-calendar: value prop can be null too

### PR DESCRIPTION
**Pull request by giacomocerquone**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Couldn't run the last one. On OSX I've got this issue: `Cannot find module 'csstype' or its corresponding type declarations`

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/wojtekmaj/react-calendar/issues/380
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

No substantial changes

In that issue the source code is visible!
Basically the "value" prop must be allowed to have a null type because it's intentionally coded that when it's null, it resets a particular UI effect.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react-calendar ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-calendar@latest/))
was missing the following properties:
1. Calendar
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 87b6cf278980c727b7da3fb2cf02c32f9225d11c
</p>

**Review by orta — APPROVED** [review]
